packages/expo-dev-launcher/package.json

```diff
@@ -30,15 +30,14 @@
   "homepage": "https://docs.expo.dev",
   "dependencies": {
     "@expo/config-plugins": "^4.0.14",
+    "expo-dev-menu": "0.9.1",
     "resolve-from": "^5.0.0",
-    "semver": "^7.3.5",
-    "expo-dev-menu": "0.9.1"
+    "semver": "^7.3.5"
   },
   "devDependencies": {
-    "@react-navigation/bottom-tabs": "^6.0.9",
-    "@react-navigation/core": "5.15.1",
-    "@react-navigation/native": "5.9.2",
-    "@react-navigation/stack": "5.14.2",
+    "@react-navigation/bottom-tabs": "^6.2.0",
+    "@react-navigation/native": "^6.0.8",
```

Comment:
will upgrading this dependency require us to vendor react-native-screens for the dev-launcher app as well?

Reply:
I didn't notice this in dev-launcher - good catch! - this will have to be reverted

Reply:
How taxing is it to vendor RN Screens for dev-launcher? We won't want to be stuck on React Navigation 5 forever, so we'll have to do this work eventually

Reply:
would prefer not to unless it's really necessary, because anything we vendor is shipped with the dev client package and bloats people's projects. is it possible to use react navigation v6 without react-native-screens?

Reply:
Yeah, it is! I think just not including the package at all disables react-native-screens integration in react-navigation 6

Reply:
oh, cool, i think this is probably a non-issue then. sorry for the confusion!!